package.json

```diff
@@ -44,7 +44,7 @@
     "serve": "*",
     "sinon": "1.17.7",
     "unexpected": "10.23.0",
-    "unexpected-documentation-site-generator": "^4.2.3",
-    "unexpected-markdown": "1.4.1"
+    "unexpected-documentation-site-generator": "4.5.0",
```

Comment:
Oh, sorry, that was an accident. Fixed in https://github.com/unexpectedjs/unexpected-sinon/commit/745bc2d74fe776fab792e32a270efa37d5c1b567